pymongo/typings.py

```diff
@@ -34,6 +34,7 @@
 _Address = Tuple[str, Optional[int]]
 _CollationIn = Union[Mapping[str, Any], "Collation"]
 _Pipeline = Sequence[Mapping[str, Any]]
+ClusterTime = Optional[Mapping[str, Any]]
```

Comment:
Personally I think we should nix the Optional here.

Reply:
I don't have a huge preference either way, so its changed :)